### PR TITLE
Fixed broken formatting, cleaned up tables, lists, etc.

### PR DESCRIPTION
**Pull request by davidwrede**
Signed-off-by: David Wrede <dwrede@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/docs/concepts-scaffolding.html.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ To begin using Scaffolding, you will need to add the appropriate `pkg_scaffoldin
     pkg_version=MY_VERSION
     pkg_scaffolding=core/scaffolding-ruby
 
-You can also use the cli command `hab plan init -s [PKG_NAME]` to initialize a scaffolding. Read more on the [CLI documentation page](/docs/reference/cli-docs/)
-* hab plan init -s [SCAFFOLDING_NAME] [PKG_NAME] 
-* Optional scaffolding names: ruby, node
+You can also use the cli command `hab plan init -s [SCAFFOLDING] [PKG_NAME]` to initialize a scaffolding. Read more on the [CLI documentation page](/docs/reference/habitat-cli/#hab-plan-init) or see how to use it in the [Build a Sample App](/tutorials/sample-app/mac/) tutorial.
```
Comment (reset) [inline www/source/docs/concepts-scaffolding.html.md]: I think this is more appropriately written: "to initialize a plan with a given scaffolding"

A Scaffolding is an artifact itself and isn't being initialized, the plan is being given power via a scaffolding

File: www/source/docs/concepts-scaffolding.html.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ To begin using Scaffolding, you will need to add the appropriate `pkg_scaffoldin
...
+
+* Optional scaffolding names: `ruby`, `node`
```
Comment (reset) [inline www/source/docs/concepts-scaffolding.html.md]: Since the scaffolding is a noun, would we maybe say "Available scaffolding" instead?

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

File: www/source/docs/run-packages-binding.html.md [hunk 3]
```diff
@@ -4,19 +4,9 @@ title: Running packages with runtime binding
...
+*Runtime binding* in Habitat refers to the ability for one service group to connect to another forming a producer/consumer relationship where the consumer can use the producer's publicly available configuration to configure it's services at runtime. A [polymorphic relationship](https://en.wikipedia.org/wiki/Polymorphism_(computer_science)) is formed between these two service groups defined by a [dynamically-typed](https://en.wikipedia.org/wiki/Duck_typing) contract where the consumer specifies a generic name of the service it is consuming along with the configuration keys it expects the producer to export.
```
Comment (tashimi) [inline www/source/docs/run-packages-binding.html.md]: This is way information overload. 

Cut this: A [polymorphic relationship](https://en.wikipedia.org/wiki/Polymorphism_(computer_science)) is formed between these two service groups defined by a [dynamically-typed](https://en.wikipedia.org/wiki/Duck_typing) contract where the consumer specifies a generic name of the service it is consuming along with the configuration keys it expects the producer to export.

File: www/source/docs/concepts-scaffolding.html.md [hunk 4]
```diff
@@ -13,9 +13,9 @@ To begin using Scaffolding, you will need to add the appropriate `pkg_scaffoldin
     pkg_version=MY_VERSION
     pkg_scaffolding=core/scaffolding-ruby
 
-You can also use the cli command `hab plan init -s [PKG_NAME]` to initialize a scaffolding. Read more on the [CLI documentation page](/docs/reference/cli-docs/)
-* hab plan init -s [SCAFFOLDING_NAME] [PKG_NAME] 
-* Optional scaffolding names: ruby, node
+You can also use the cli command `hab plan init -s [SCAFFOLDING] [PKG_NAME]` to initialize a plan with a given scaffolding. Read more on the [CLI documentation page](/docs/reference/habitat-cli/#hab-plan-init) or see how to use it in the [Build a Sample App](/tutorials/sample-app/mac/) tutorial.
```
Comment (tashimi) [inline www/source/docs/concepts-scaffolding.html.md]: Up on line 5 we are duplicating the content of the below list of available & coming scaffoldings. Suggest cutting that.

File: www/source/docs/container-orchestration-mesos.html.md [hunk 5]
```diff
@@ -22,23 +22,29 @@ You can create native Mesos containers from Habitat packages by following these
 
 2. Install or [build](/docs/create-packages-build) the Habitat package from which you want to create a Marathon application, for example:
 
-       hab pkg install yourorigin/yourpackage
+    ```
```
Comment (tashimi) [inline www/source/docs/container-orchestration-mesos.html.md]: We should port this entire article into a blog with the date is was published. Doesn't read like docs. Reads like something written before we had a blog to put it in.

**Comment by tashimi** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by tashimi** [comment]
tenor-261285044

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/277578822).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/277583147).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/277578822).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/277583147).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**